web/src/components/user/UserAPITab.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <Panel>
+    <div class="flex flex-row border-b mb-4 pb-4 items-center dark:border-wp-background-100">
+      <div class="ml-2">
+        <h1 class="text-xl text-wp-text-100">{{ $t('user.settings.api.api') }}</h1>
+        <p class="text-sm text-wp-text-alt-100">{{ $t('user.settings.api.desc') }}</p>
+      </div>
+    </div>
+
+    <div class="mt-2 mb-4">
+      <div class="flex items-center mb-2">
+        <div class="flex items-center text-wp-text-100 font-bold">
+          <label>{{ $t('user.settings.api.token') }}</label>
+        </div>
+        <Button class="ml-4" :text="$t('user.settings.api.reset_token')" @click="resetToken" />
+      </div>
```

Comment:
```suggestion
      <div class="flex items-center mb-2">
        <div class="flex flex-row items-center text-wp-text-100 font-bold">
          <label>{{ $t('user.settings.api.token') }}</label>
        </div>
        <Button class="ml-auto" :text="$t('user.settings.api.reset_token')" @click="resetToken" />
      </div>
```

What do you think about aligning the button to the right?

![image](https://github.com/woodpecker-ci/woodpecker/assets/3391958/5e9de72a-dae7-4df4-982e-595b35792425)